frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,65 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  margin: "0px",
+  width: "100%",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+  fontSize: "24px",
```

Comment:
Can we use the typography component for this?

Reply:
left a Q on figma to confirm icon font size